src/listeners/ready.ts

```diff
@@ -0,0 +1,14 @@
+import { Listener } from 'discord-akairo';
+
+class Ready extends Listener {
+    constructor() {
+        super('ready', {
+            emitter: 'client',
+            event: 'ready'
+        });
+    }
+    exec() {
+        console.log('I\'m ready!');
```

Comment:
Aren't there any inline way to define the listener?

Reply:
most likely